src/common/transformations/include/transformations/common_optimizations/transpose_sinking_utils.hpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include <openvino/opsets/opset9.hpp>
+#include <openvino/pass/pattern/op/or.hpp>
+#include <transformations/utils/utils.hpp>
+#include <utility>
+
+#include "itt.hpp"
+#include "openvino/op/util/op_types.hpp"
+#include "openvino/opsets/opset9.hpp"
+#include "openvino/pass/pattern/op/label.hpp"
+#include "openvino/pass/pattern/op/wrap_type.hpp"
+#include "openvino/util/common_util.hpp"
+#include "openvino/util/log.hpp"
+
+namespace transpose_sinking {
+
+using namespace ov;
+using namespace ov::opset9;
```

Comment:
are you sure about `using ....` in header file? it can be dangerous 
I would recommend do not use `using` in header files at all, in `cpp` only